README.md

```diff
@@ -261,73 +259,350 @@ The SDK provides a simple interface for all the [media APIs mentioned here](http
 Accepts an object specifying the parameters to be used to list and search files. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/list-and-search-files) can be passed as it is with the correct values to get the results.
 
 ```cs
-List<ListAPIResponse> resp = await imagekit
-    .Skip(0)
-    .Limit(10)
-    .ListFilesAsync();
+ResponseMetaData resp = await imagekit.GetFileListRequestAsync();
 ```
 
-You can also use advanced [searching](https://docs.imagekit.io/api-reference/media-api/list-and-search-files#advanced-search-queries) and sorting using `SearchQuery` and `Sort` function.
 
-```cs
-List<ListAPIResponse> resp = await imagekit
-    .Skip(0)
-    .Limit(3)
-    .Sort("DESC_SIZE")
-    .SearchQuery("tags IN [\"tag1\"]")
-    .ListFilesAsync();
-```
+
 
 **2. Get File Details**
 
 Accepts the file ID and fetches the details as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/get-file-details).
 
 ```cs
-ListAPIResponse resp = await imagekit.GetFileDetailsAsync(fileId);
+Task<ResponseMetaData> resp = await imagekit.GetFileDetailsAsync(fileId);
 ```
 
-**3. Get File Metadata**
+**3. Get File Versions**
 
-Accepts the `fileId` or `URI` and fetches the metadata as per the [API documentation here](https://docs.imagekit.io/api-reference/metadata-api/get-image-metadata-for-uploaded-media-files).
+Accepts the file ID and fetches the details as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/get-file-versions).
+
+```.net
+String fileId = "62a04834c10d49825c6de9e8";
+ResponseMetaData resultFileVersions = ImageKit.getFileVersions(fileId);
```

Comment:
In all the above examples so far, we have small `i` i.e. `imageKit` instead of `ImageKit`. Let's pick one and be consistent throughout all the documentation and examples.